tools/validate_spec.py

```diff
@@ -101,6 +110,32 @@ def cli(argv: List[str]) -> Union[int, str]:
     except RecordFluxError as e:
         return f"invalid identifier: {e}"
 
+    checksum_functions = None
+    if args.checksum_functions is not None:
+        try:
+            checksum_module = importlib.import_module(str(args.checksum_functions))
+        except ImportError as e:
+            return (
+                f"The provided module {args.checksum_functions} cannot be "
+                f"imported. Make sure the module name is provided as "
+                f"package.module and not as a file system path. {e}"
+            )
+
+        try:
+            all_checksum_functions = checksum_module.checksum_functions  # type: ignore
```

Comment:
Only the specific type problem should be ignored: `# type: ignore[attr-defined]`.

Reply:
Fixed in 6f94ec1